met-web/src/components/engagement/banner/BannerWithImage.tsx

```diff
@@ -13,6 +13,13 @@ const BannerWithImage = ({ savedEngagement, children }: BannerProps) => {
     const [imageError, setImageError] = useState(false);
     const isSmallScreen = useMediaQuery((theme: Theme) => theme.breakpoints.down('sm'));
 
+    const EngagementDateString = () => {
```

Comment:
might as well be a variable and not a function since the output is not changing on a certain input. just const = EngagementDateRange = `Engagement dates ...`. And in variable naming the 'String' suffix doesn't add any meaning since it's obviously a string

Reply:
sounds good